Headers/diaglist_mod.F90

```diff
@@ -81,11 +81,13 @@ MODULE DiagList_Mod
   !=========================================================================
   ! Configurable Settings Used for Diagnostic Names at Run-time
   !=========================================================================
-  CHARACTER(LEN=5),  PUBLIC  :: RadWL(3)     ! Wavelengths in radiation menu
-  CHARACTER(LEN=4),  PUBLIC  :: RadOut(12)   ! Names of RRTMG outputs (tags)
-  INTEGER,           PUBLIC  :: nRadOut      ! # of selected RRTMG outputs
-  LOGICAL,           PUBLIC  :: IsFullChem   ! Is this a fullchem simulation?
-  CHARACTER(LEN=10), PUBLIC  :: AltAboveSfc  ! Alt for O3, HNO3 diagnostics
+  CHARACTER(LEN=5),  PUBLIC  :: RadWL(3)      ! Wavelengths in radiation menu
+  CHARACTER(LEN=4),  PUBLIC  :: RadOut(12)    ! Names of RRTMG outputs (tags)
+  INTEGER,           PUBLIC  :: nRadOut       ! # of selected RRTMG outputs
+  LOGICAL,           PUBLIC  :: isCarbonCycle ! Is this a carbon cycle sim?
+  LOGICAL,           PUBLIC  :: isFullChem    ! Is this a fullchem simulation?
+  LOGICAL,           PUBLIC  :: isMercury     ! Is this a mercury simulation?
+  CHARACTER(LEN=10), PUBLIC  :: AltAboveSfc   ! Alt for O3, HNO3 diagnostics
```

Comment:
Added isCarbonCycle and isMercury so that we can use these flags in state_diag_mod.F90 where we set the metadata.